internal/commands/create.go

```diff
@@ -224,6 +226,62 @@ func getRegoFilePaths(path string) ([]string, error) {
 	return regoFilePaths, nil
 }
 
+func parsePolicies(policyPaths []string, libraryPaths []string) ([]*regoPolicy, error) {
+	policies, err := loadPolicies(policyPaths)
+	if err != nil {
+		return nil, err
```

Comment:
nit: we should wrap all errors so the entire chain of events is bubbled to the caller